lib/ndr_import/file/xml.rb

```diff
@@ -16,32 +16,87 @@ def initialize(*)
         super
 
         @pattern_match_xpath = @options['pattern_match_record_xpath']
+        @xml_file_metadata   = @options['xml_file_metadata']
```

Comment:
Maybe document `xml_file_metadata` in `docs/xml-mappings.md`?

Reply:
Thanks @bshand - good thinking. 77afde5